web/controllers/admin_controller.ex

```diff
@@ -319,6 +319,7 @@ defmodule ExAdmin.AdminController do
   end
 
   defp to_integer(:string, string), do: string
+  defp to_integer(:id, string), do: string
```

Comment:
Hello Steve, 

Sorry I was away for couple of days.
I am looking into tests. And going to add tests for this one.